xpra/server/window/window_video_source.py

```diff
@@ -328,8 +330,8 @@ def video_context_clean(self):
         if csce or ve:
             if DEBUG_VIDEO_CLEAN:
                 log.warn("video_context_clean() for wid %i: %s and %s", self.wid, csce, ve)
-                import traceback
-                traceback.print_stack()
+                # import traceback
+                # traceback.print_stack()
```

Comment:
3d227dc50883d1a2f4a94575704a2ec850df776d

Reply:
I took this out because when looking at logs this appeared to be a stack trace from an exception when it actually wasn't and kept masking the actual problems when skimming through logs. Either way works - this one isn't critical